lib/dice_bag/command.rb

```diff
@@ -25,15 +32,14 @@ def write(template_name, params = {})
     end
 
 
-    def generate_all_templates
+    def generate_all_templates(force=false)
       AvailableTemplates.all.each do |template|
-        generate_template(template)
+        generate_template(template, force)
       end
     end
 
-    def generate_template(default_template)
-      default_template.assert_existence
-      default_template.create_file
+    def generate_template(default_template, force=false)
+      copy_file default_template.file, default_template.destination, force: force
```

Comment:
does copy_file puts some message when it just copies over and there was no file to overwrite?
I just want to have some notification that things indeed worked.